app/src/main/java/org/android/go/sopt/home/GalleryFragment.kt

```diff
@@ -23,6 +25,17 @@ class GalleryFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding.vpGallery.adapter = GalleryAdapter().apply {
+            setGalleryItemList(
+                listOf(
+                    R.drawable.img_keroro_afro,
+                    R.drawable.img_keroro_king,
+                    R.drawable.img_kururu,
+                    R.drawable.img_zeroro,
+                    R.drawable.img_tamama_giroro
+                )
+            )
+        }
     }
 
     override fun onDestroyView() {
```

Comment:
```suggestion
    override fun onDestroyView() {
        _binding = null
        super.onDestroyView()
    }
```
메모리 누수 방지를 위해서 _binding을 null로 초기화하고 난뒤, onDestroyView()를 호출합시당